src/main/java/entity/Item.java

```diff
@@ -0,0 +1,50 @@
+package entity;
+
+public abstract class Item {
+    private final String name; // the name of the entity.Item
+    private int num; // the number of items in entity.Item
+
+    /**
+     * Creates a new entity.Item object.
+     * This constructor takes in a single String as and a single int as arguments..
+     * @param name the name of the entity.Item.
+     * @param num the number of items in the entity.Item object.
+     */
+    public Item(String name, int num) {
+        this.name = name;
```

Comment:
comment was marked as resolved but I'm not sure if anything changed, this line should be changed to 
`this(name);
this.num = num;`

Reply:
oops my bad, fixed